### PR TITLE
search bars

### PR DESCRIPTION
**Pull request by silchyn**
- [DEMO LINK](https://silchyn.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://silchyn.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+  line-height: 22px;
+  font-size: 16px;
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 800;
```
Comment (VETALEK) [inline src/style.css:32]:
```suggestion
  width: 100%;
  height: 70px;
  padding-left: 62px;
  
  line-height: 22px;
  font-size: 16px;
  font-family: "Avenir", sans-serif;
  color: #3d4e61;
  font-weight: 800;
  
  background: #fff;
  border: 1px solid #e1e7ed;
  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
  border-radius: 4px;
```
better group rules by theme, like {
  position

  block model

  typography

  appearance

  other
}

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,40 @@
...
+        class="icon"
+      >
+      <input
+        type="search"
+        data-qa="keypress"
+        class="input"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+    <form
```
Comment (VETALEK) [inline src/index.html:32]:
```suggestion
    </form>
    
    <form
```
that would be perfect 🤩

**Review by VETALEK — APPROVED** [review]
Well done 💯

File: src/style.css (hunk 1 above)
Comment (silchyn) [reply src/style.css:32]: Understood, thanks!

**Pull request closed without merging**